package/lib-tensorflow_cc-shared-1.4.0/.cm/meta.json

```diff
@@ -0,0 +1,98 @@
+{
+  "customize": {
+    "extra_dir": "", 
+    "git_src_dir": "tensorflow_cc", 
+    "install_env": {
+      "CMAKE_CONFIG": "Release", 
+      "PACKAGE_AUTOGEN": "NO", 
+      "PACKAGE_BUILD_TYPE": "cmake", 
+      "PACKAGE_CONFIGURE_FLAGS": "", 
+      "PACKAGE_CONFIGURE_FLAGS_ANDROID": "-DBUILD_SHARED_LIBS=OFF", 
+      "PACKAGE_CONFIGURE_FLAGS_LINUX": "", 
+      "PACKAGE_CONFIGURE_FLAGS_WINDOWS": "", 
+      "PACKAGE_GIT": "YES", 
+      "PACKAGE_GIT_CHECKOUT": "lib-tensorflow_cc-shared-1.4.0",
+      "PACKAGE_PATCH": "YES", 
+      "PACKAGE_SKIP_CLEAN_INSTALL": "NO", 
+      "PACKAGE_SKIP_CLEAN_OBJ": "YES", 
+      "PACKAGE_SKIP_CLEAN_PACKAGE": "NO", 
+      "PACKAGE_SKIP_CLEAN_SRC_DIR": "YES", 
+      "PACKAGE_SKIP_CMAKE_TARGET": "NO", 
+      "PACKAGE_SUB_DIR": "tensorflow_cc", 
+      "PACKAGE_SUB_DIR1": "tensorflow_cc/tensorflow_cc", 
+      "PACKAGE_URL": "https://github.com/ctuning/tensorflow_cc.git"
+    }, 
+    "version": "1.4"
+  }, 
+  "deps": {
+    "bazel": {
+      "add_to_path": "no", 
+      "local": "no", 
+      "name": "Bazel tool", 
+      "sort": 40, 
+      "tags": "tool,bazel", 
+      "version_from": [
+        0, 
+        5
+      ], 
+      "version_to": [
```

Comment:
We should be able to build at least with current Bazel 0.6 and 0.7...

Reply:
I've increased version to 0.7 but we have only bazel 0.6.1 in ck_env, so I've checked if it is built with this version only.

Reply:
> Should we add package ck_env:package:tool-bazel-0.7.*-linux now
Yes, please: https://github.com/dividiti/ck-tensorflow-private/issues/10